src/app/components/profile-page/profile-page.component.html

```diff
@@ -1,20 +1,45 @@
-<main class="container px-1">
-  <ng-container *ngIf="!deleteMode else removableItems">
-    <div *ngFor="let image of images" class="img-container m-1">
-      <a [routerLink]="['image']" [queryParams]="{imageName: image.imageName, imageUrl: image.imageUrl }"
-        queryParamsHandling="merge">
-        <img class="item-img w-100 h-100" [src]="image.imageUrl" alt="Picture">
-      </a>
-    </div>
-  </ng-container>
-  <ng-template #removableItems>
-    <div class='image-container' *ngFor="let image of images" (click)="selectFile(image.imageName)">
-      <img [ngClass]="checkSelectedImage(image.imageName) ? 'fade-image': ''" class="item-img w-100 h-100 p-1"
-        [src]="image.imageUrl" alt="Picture">
-      <div [ngClass]="checkSelectedImage(image.imageName) ? 'display-icon': ''" class="download-link fa-3x">
-        <fa-icon [ngClass]="checkSelectedImage(image.imageName) ? 'position-icon': ''" [icon]="faCheck">
-        </fa-icon>
+<main class="container">
+  <div class="my-4 d-flex align-items-center">
+    <div><app-profile-indicator></app-profile-indicator></div>
+
+    <div *ngIf="!deleteMode" ngbDropdown class="d-inline-block ml-auto">
+      <button class="img-button border-0 ml-auto px-2 mr-2" id="post-dropdown" ngbDropdownToggle>
+        <fa-icon [icon]="faEllipsisV"></fa-icon>
+      </button>
+
+      <div ngbDropdownMenu aria-labelledby="post-dropdown" placement="right-bottom">
+        <button ngbDropdownItem (click)="toggleDelete()" [disabled]="isDeleting">
+          <div *ngIf="isDeleting" class="spinner-border spinner-border-sm mr-2" role="status">
+            <span class="sr-only">Loading...</span>
+          </div>
+
+          Delete Posts
+        </button>
+
+        <button ngbDropdownItem (click)="logOut()">Log out</button>
       </div>
     </div>
-  </ng-template>
-</main>
\ No newline at end of file
+  </div>
+
+  <div class="images-container">
+    <ng-container *ngIf="!deleteMode else removableItems">
+      <div *ngFor="let image of images" class="img-container m-1">
+        <a [routerLink]="['image']" [queryParams]="{imageName: image.imageName, imageUrl: image.imageUrl }"
+          queryParamsHandling="merge">
+          <img class="item-img w-100 h-100" [src]="image.imageUrl" alt="Picture">
+        </a>
+      </div>
+    </ng-container>
+
```

Comment:
Change the ng-container to ng-template since it not needed to use ng-container

Reply:
We can use `ng-template` then the syntax will change to `<ng-template [ngIf]="!deleteMode" [ngIfElse]="removableItems">`. 
However this is also valid. Here we are using `ng-container` because we cant have the `*ngIf` and `*ngFor` in the same div. And in my opinion we are already familliar with the `*` syntax used by `ng-container` which is nice. 

We can discuss it as a team if you would still like me to change it.